docdoku-server/docdoku-server-web/src/main/java/com/docdoku/server/rest/dto/LightPartRevisionDTO.java

```diff
@@ -0,0 +1,44 @@
+package com.docdoku.server.rest.dto;
```

Comment:
Do we really need such class?
Why not use the classic one and not fill the unused property?